MAUI/PDF-Viewer/Add-Remove-Modify-Annotations.md

```diff
@@ -171,4 +171,21 @@ private void OnAnnotationEdited(object sender, AnnotationEventArgs e)
     Debug.WriteLine("The {0} is edited successfully", editedAnnotation.Name);
 }
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+### Adding custom information to a annotation
+
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to assign custom information to annotations. The `CustomData` property is utilized to store additional information about each annotation instance for reference. However, it's essential to note that these data are solely intended for reference purposes and will not be displayed in the PdfViewer interface.
```

Comment:
<img width="387" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/b427d52d-80e6-4456-9ca3-699211aff79e">

Please mention the PDF Viewer similar to other places.